### PR TITLE
Updated AssetList Component w/ mock data

### PR DESCRIPTION
**Pull request by joeadams04**
Removed 'Edited recently'
Updated AssetList Container
New AssetItem Component
Layout fixes

Currently using mock data to populate table

### CODE REVIEW

File: src/containers/AssetList.js [hunk 1]
```diff
@@ -1,62 +1,116 @@
...
-                User behaviour for Research Dashboard
-              </Link>
-            </TableRowColumn>
-            <TableRowColumn>Complete</TableRowColumn>
-            <TableRowColumn>University Information Services (UIS)</TableRowColumn>
-            <TableRowColumn>11/01/2018</TableRowColumn>
-          </TableRow>
-        </TableBody>
-      </Table>
+  	<div className="App-main">
```
Comment (rjw57) [inline src/containers/AssetList.js]: Can we use an indent of 2 spaces here to be consistent with the rest of the code?

File: src/style/App.css [hunk 2]
```diff
@@ -39,3 +47,8 @@
   font-size: 14px;
   font-weight: 500;
 }
+
+.Asset-item {
+  display: block;
+  width: 100%;
+}
```
Comment (rjw57) [inline src/style/App.css]: Can we move this to components/AssetListItem.css and import it from AssetListItem.js? Also, I don't see where this class is actually applied to an element. Is it implicit?

**Comment by rjw57** [comment]
The [test suite output](https://travis-ci.org/uisautomation/iar-frontend/builds/333154920) includes a lot of warnings. Can these be addressed in this PR?

```
 PASS  src/containers/AssetForm.test.js (5.425s)
 PASS  src/containers/AssetList.test.js (5.804s)
  ● Console
    console.error node_modules/fbjs/lib/warning.js:33
      Warning: Each child in an array or iterator should have a unique "key" prop.
      
      Check the render method of `AssetList`. See https://fb.me/react-warning-keys for more information.
          in AssetListItem (at AssetList.js:100)
          in AssetList (at AssetList.test.js:8)
          in MuiThemeProvider (at testutils.js:13)
          in Router (created by MemoryRouter)
          in MemoryRouter (at testutils.js:13)
```

File: src/containers/AssetList.js [hunk 3]
```diff
@@ -65,52 +65,50 @@ const assetData = [
 ];
 
 const TITLES = {
-  '/assets/dept': 'Assets: My department',
-  '/assets/all': 'Assets: All',
+	'/assets/dept': 'Assets: My department',
+	'/assets/all': 'Assets: All',
```
Comment (rjw57) [inline src/containers/AssetList.js]: I think the consistent indent size should actually be 2 to match, e.g. Sidebar.js, Static.js, AssetFormHEader.js. Maybe add a [*.js] section to .editorconfig to this effect?

**Comment by joeadams04** [comment]
Fixed indentation issues throughout AssetList.

Regarding the styling, the class was used when I was playing around with the layout, however in the end wasn't needed. I've removed it now. From now on, yes we should probably create separate style sheets for each component.

**Comment by rjw57** [comment]
Thanks, as you can see, I think the indent should be 2 spaces, not 4. Also, are your changes the source of the warnings in the test suite output?

File: src/containers/AssetList.js (hunk 3 above)
Comment (joeadams04) [reply src/containers/AssetList.js]: Spaces or tabs?

Comment (joeadams04) [reply src/containers/AssetList.js]: Ignore that last comment, changes my tab size to 2 spaces.

Comment (rjw57) [reply src/containers/AssetList.js]: Nooo..... spaces! If you use a tab size of 2, GitHub will still display them as 8.

File: src/containers/AssetList.js [hunk 4]
```diff
@@ -97,7 +97,7 @@ const AssetList = ({ match }) => (
         >
           {assetData.map( (asset, index) => (
             <AssetListItem
-              index={asset.index}
+              key={index}
```
Comment (rjw57) [inline src/containers/AssetList.js]: When we have a real API, this should be asset.id or similar. For the moment, this is fine.

File: src/containers/AssetList.js [hunk 5]
```diff
@@ -1,58 +1,110 @@
-import React from 'react';
+import React, {Component} from 'react';
 import { Link } from 'react-router-dom'
```
Comment (rjw57) [inline src/containers/AssetList.js]: In testing I get the following:

```
./src/containers/AssetList.js
  Line 1:  'Component' is defined but never used  no-unused-vars
  Line 2:  'Link' is defined but never used       no-unused-vars
```

It's so trivial that I'll just update the PR to remove these and then accept.

**Pull request merged**